optimum/onnxruntime/trainer.py

```diff
@@ -134,6 +139,30 @@
 SCALER_NAME = "scaler.pt"
 
 
+class ModuleWithLoss(nn.Module):
```

Comment:
`ModuleWithLoss` as a wrapper for `torch.nn.module` subclass, can you add a `module` property so that the [`unwrap_model`](https://github.com/huggingface/transformers/blob/bb5a2f2fc30985841289207b9f1f7765d8abc4e0/src/transformers/modeling_utils.py#L3548) could be compatible? I believe [`ORTModule` did the same](https://github.com/microsoft/onnxruntime/blob/6320decf048914009a42528914986eabedb1e900/orttraining/orttraining/python/training/ortmodule/ortmodule.py#L153)